patches/jruby.diff

```diff
@@ -51,66 +51,73 @@ index 50a155d..0234ef7 100644
  
  @CoreClass("Kernel")
  public abstract class KernelNodes {
-@@ -1985,4 +1986,49 @@ public abstract class KernelNodes {
+@@ -1985,4 +1986,56 @@ public abstract class KernelNodes {
  
      }
  
-+    @CoreMethod(names = "read_ts_reg_start", needsSelf=false)
-+    public abstract static class ReadTSRegStartNode extends CoreMethodArrayArgumentsNode {
++    @CoreMethod(names = "read_core_cycles", needsSelf=false)
++    public abstract static class ReadCoreCyclesNode extends CoreMethodArrayArgumentsNode {
 +
 +        private static final BigInteger negAdjust = BigInteger.ONE.shiftLeft(64); // 2^64
```

Comment:
Note that this is the first line of a new block. Looks OK to me, unless I am mistaken.